docs/configuration.md

```diff
@@ -17,6 +17,37 @@ the certificate for authenticating the service account on pod.
 - `app` contains information about the extension app that the bot will use like 
 the appId, the private key or the certificate for authenticating the extension app.
 - `ssl` contains trustStore and trustStore password for SSL communication.
+- `datafeed` contains information of the datafeed service to be used by the bot.
+- `retry` contains information for retry mechanism to be used by the bot.
+
+### Retry Configuration
+The retry mechanism used by the bot will be configured by these following properties:
```

Comment:
How is that going to play with the BOC? (i.e should put in place such a complex configuration if we plan to delegate some of that logic to the BOC)?